app/components/pages/Dashboard/Dashboard.js

```diff
@@ -1,90 +1,78 @@
 import React from 'react'
 import styled from 'styled-components'
-import { Link } from 'react-router-dom'
-import { Box } from 'grid-styled'
+import { Box, Flex } from 'grid-styled'
 import { Button } from '@pubsweet/ui'
-
+import { th } from '@pubsweet/ui-toolkit'
 import media from '../../global/layout/media'
-import { FormH2, FormH3 } from '../../ui/atoms/FormHeadings'
-import Paragraph from '../../ui/atoms/Paragraph'
-import ManuscriptStatus from '../../ui/atoms/ManuscriptStatus'
+import SmallParagraph from '../../ui/atoms/SmallParagraph'
+import ExternalLink from '../../ui/atoms/ExternalLink'
 import StickyFooter from '../../ui/atoms/StickyFooter'
+import Tabs from '../../ui/molecules/Tabs'
+import DashboardList from '../../ui/molecules/DashboardList'
+import { ALL_MANUSCRIPTS } from './oporations'
+import ManuscriptsQuery from './ManuscriptsQuery'
 
-/* Temporary dashboard view pending receipt of designs */
-
-const BoxHiddenFromMobile = styled(Box)`
+const RightAlignedButton = styled(Box)`
+  margin: ${th('space.3')} 0 ${th('space.1')};
+  text-align: right;
   display: none;
   ${media.mobileUp`display: block`};
 `
 
+const DashboardPanel = styled(Tabs.Panel)`
+  min-height: 300px;
+`
+
 const MobileOnlyStickyFooter = styled(StickyFooter)`
   display: block;
   ${media.mobileUp`display: none;`};
 `
 
-const Header = styled.th`
-  text-align: left;
-  padding: 3px 6px;
+const EmptyListMessage = styled.div`
```

Comment:
Could we make this a styled `p` (or `Paragraph`) just for semantics? Off the top of my head, I can't see a downside, but perhaps there is one?

Reply:
Ah, just seen `CentredSmallParagraph` below. Could we use this again & perhaps this `div` around it could be responsible only for adding the `margin-top` value?

Better yet, use a grid-styled element? :smile: